runtime/hydradx/src/lib.rs

```diff
@@ -207,6 +207,7 @@ impl Contains<Call> for CallFilter {
 			// Lookup::lookup() is not necessary thanks to IdentityLookup
 			if dest == &Omnipool::protocol_account()
 				&& *currency_id == <Runtime as pallet_omnipool::Config>::HubAssetId::get()
+				|| Omnipool::exists(*currency_id)
```

Comment:
this filters all transfers to any account if asset is in the omnipool.

Reply:
this also allows HDX to be transfered. We need to filter out balances transfer too.

Reply:
Nice catch, thanks. Fixed.